client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -471,7 +498,7 @@ class TransferDomainStep extends React.Component {
 					},
 				} );
 
-				if ( callback ) {
+				if ( isFunction( callback ) ) {
```

Comment:
Hmm, you encountered a bug with this?

Reply:
Yes...when you click on the "refresh status" link on the Precheck screen, this throws an error.

Reply:
:man_facepalming: